homeassistant/components/mqtt/strings.json

```
@@ -27,7 +27,10 @@
           "tls_insecure": "Ignore broker certificate validation",
           "protocol": "MQTT protocol",
           "set_ca_cert": "Broker certificate validation",
-          "set_client_cert": "Use a client certificate"
+          "set_client_cert": "Use a client certificate",
+          "transport": "MQTT Transport",
```

If "MQTT protocol" isn't using Title Case, maybe this shouldn't either?
```suggestion
          "transport": "MQTT transport",
```